From 7bb6ef5bb43f565286dffe685bdb5dc2c569f64f Mon Sep 17 00:00:00 2001
From: Swee Yee Fonn <swee.yee.fonn@intel.com>
Date: Wed, 4 Nov 2020 19:01:24 +0800
Subject: [PATCH 1/2] Update boot options for keyboxd service

Update keyboxd service boot for support keybox
provisioning over vsocket.

Tracked-On: OAM-95582
Change-Id: Id072c8ca0d7f2a0919419585a4d7ff54c6b2809d
Signed-off-by: Swee Yee Fonn <swee.yee.fonn@intel.com>
---
 groups/trusty/true/init.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/groups/trusty/true/init.rc b/groups/trusty/true/init.rc
index 39d7c85..45bf53a 100644
--- a/groups/trusty/true/init.rc
+++ b/groups/trusty/true/init.rc
@@ -14,7 +14,7 @@ service storageproxyd /vendor/bin/storageproxyd -d /dev/trusty-ipc-dev0 -p /data
 on boot
     start keyboxd

-service keyboxd /vendor/bin/keybox_provisioning -d /dev/trusty-ipc-dev0 -p /dev/block/by-name/teedata
+service keyboxd /vendor/bin/keybox_provisioning -d /dev/trusty-ipc-dev0 -p /dev/block/by-name/teedata -v 5392
     user system
     group system
     oneshot
--
2.17.1

